napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -64,27 +101,49 @@ def __init__(
         enabled: bool = True,
         installed: bool = False,
         npe_version=1,
+        versions_conda: List[str] = None,
+        versions_pypi: List[str] = None,
     ) -> None:
         super().__init__(parent)
+        self.url = url
+        self._versions_conda = versions_conda
+        self._versions_pypi = versions_pypi
         self.setup_ui(enabled)
         self.plugin_name.setText(package_name)
+
+        if len(versions_pypi) > 0:
+            self._populate_version_dropdown('PyPI')
+        else:
+            self._populate_version_dropdown('Conda')
+
         self.package_name.setText(version)
-        self.summary.setText(summary)
-        self.package_author.setText(author)
+        if summary:
+            self.summary.setText(summary + '<br />')
+        if author:
+            self.package_author.setText(author)
+        self.package_author.setWordWrap(True)
         self.cancel_btn.setVisible(False)
 
-        self.help_button.setText(trans._("Website"))
-        self.help_button.setObjectName("help_button")
         self._handle_npe2_plugin(npe_version)
 
         if installed:
+            if is_conda_package(package_name):
+                self.source.setText('Conda')
```

Comment:
coverage? 
Or switch from using string literaly to global constant?

Reply:
@Czaki you mean using a global constant called `CONDA = 'Conda'` ??

Reply:
Yes, to prevent any typo related problems.